src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationTracker.cs

```diff
@@ -709,7 +709,7 @@ public CompilationInfo(Compilation compilation, bool hasSuccessfullyLoaded)
 
```

Comment:
The // PROTOTYPE two lines above seems to have been around for a long time, so I didn't touch it.